Cargo.toml

```diff
@@ -9,7 +9,6 @@ documentation = "https://docs.rs/bitcoin/"
 description = "General purpose library for using and interoperating with Bitcoin and other cryptocurrencies."
 keywords = [ "crypto", "bitcoin" ]
 readme = "README.md"
-exclude = ["./test_data"]
```

Comment:
Why? AFAIR it was previously excluded so that people using the library don't have to download the test data and development happens on top of cloned git repo anyway. Is there a new argument?

Reply:
Ah, during rebase the `test_data` directory came back into existence, thanks for flagging this. To answer your question, after I force push, the `test_data` directory will no longer exist, patch 1 of this PR moves its contents to `tests/data` - hence removal of the exclude line.

Reply:
In that case `exclude` should be changed, not removed. Am I still missing something?

Reply:
We don't think we need an explicit exclude for `tests/` since its the standard directory name for integration tests.

Reply:
Excuse the late reply, missed your comment previously.

Reply:
Yes, `tests/` is standard directory but `test/data` may be just too much for non-crate-developers. (But perhaps even remove `tests/`?)

Reply:
Oh, on further reading I think you are correct. I'll exclude `tests/`. Thanks.